src/ddmd/constfold.d

```diff
@@ -1226,7 +1226,7 @@ extern (C++) UnionExp Index(Type type, Expression e1, Expression e2)
         uinteger_t i = e2.toInteger();
         if (i >= es1.len)
         {
-            e1.error("string index %llu is out of bounds [0 .. %llu]", i, cast(ulong)es1.len);
+            e1.error("string index %llu is out of bounds `[0 .. %llu]`", i, cast(ulong)es1.len);
```

Comment:
I thought about that, and decided against it:

1. It isn't really code
2. Numbers are quite distinct anyway
3. It's done this way throughout the error message strings